### DISCUSSION TITLE
After changing default in Chrome to Local Searxng the query (%s) doesn't pass the search query through

### DISCUSSION THREAD

**Discussion by MarqBarq** [searxng/searxng#4030] [General]
Greetings. First, very new to SearXNG. Local install on docker of Synology. The site works well and I have a custom URL all proxied.

I set my new server as default in Chrome.  HTTP://localsearch.com/search?%s  

This is set as the default, and it should mean that if I type something into the chrome bar it should pass that to the above URL. What I get, for example, if I search for "fish" in the chrome bar, It returns the home page of my SearXNG page... not the results page for fish. 
 
My buddy that turned me on to SearXNG says, "It works for him". 

What am I missing? Thank you in advance.

**Comment by vojkovic** [comment]
Looks like you're missing the `q=` param in the url.

Should be: `http://localsearch.com/search?q=%s`
